pkg/network/setup/podnic.go

```diff
@@ -57,7 +58,12 @@ func newPhase2PodNIC(vmi *v1.VirtualMachineInstance, network *v1.Network, iface
 		return nil, err
 	}
 
-	ifaceLink, err := link.DiscoverByNetwork(podnic.handler, podnic.vmi.Spec.Networks, *podnic.vmiSpecNetwork, vmi.Status.Interfaces)
+	ifaceStatus := vmispec.LookupInterfaceStatusByName(vmi.Status.Interfaces, network.Name)
```

Comment:
If this change is not mandatory, please do it on follow-up PR.

Reply:
> It seem the current implementation already consume podIfaceName 7578195

Yes, for the primary interface.
This PR builds on the same mechanism for all interfaces.

> Question, does this change compatible with ordinal iface names (eth1,eth2)?

Yes, as VMI controller is aware of the naming scheme and propagates it to virt-handler and virt-launcher.

Reply:
This thread can be resolved.